vllm/model_executor/models/minicpmv.py

```diff
@@ -407,10 +1201,14 @@ def init_llm(self, config, cache_config, quant_config):
             return MiniCPMForCausalLM(config,
                                       cache_config=cache_config,
                                       quant_config=quant_config)
-        else:
+        elif self.version == 2.5:
```

Comment:
Can we detect the model version without an explicit version number? (as per my comment in https://github.com/vllm-project/vllm/issues/6814)